packages/components/app/styles/components/side-nav.scss

```diff
@@ -0,0 +1,205 @@
+  --token-side-nav-selected-surface-default: #656a76;
+  --token-side-nav-selected-surface-hover: #3b3d45;
+  --token-side-nav-selected-surface-active: #656a76;
+}
+
+
+// * SideNav Parent (wrapper) component
+// -------------------------------------------------------------------
+
+.hds-side-nav {
+  display: flex;
+  flex-direction: column;
+  width: 280px;
+  height: 100vh;
+  color: var(--token-side-nav-foreground-primary); // Delete?
+  background: var(--token-side-nav-surface-default);
+}
+
+// Child "wrapper" blocks & child components:
+
+// * Sub-component: Header
+// ---------------------------------------
+.hds-side-nav__header {
+  display: flex;
+  gap: 8px; // TODO: replace with margins
+  align-items: center;
+  padding: 12px;
+  background: var(--token-side-nav-surface-default); // Q: Should this be added here or should it just be transparent?
+
+  .hds-side-nav__header-logo {
+    margin-right: auto; // Q: Make more generic?
+  }
+
+  // TEMP HACK ONLY! (TODO: replace)
+  .hds-disclosure__toggle:not(:hover) * {
+    color: var(--token-side-nav-foreground-primary) !important;
+  }
+}
+
+.hds-side-nav__header-logo {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  min-width: 48px;
+  height: 48px;
+  padding: 4px;
+  color: var(--token-side-nav-foreground-primary);
+  background: var(--token-side-nav-surface-default); // Q: Should this be added here or should it just be transparent?
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  &:focus,
+  &.mock-focus {
+    @include hds-focus-ring-with-pseudo-element();
+  }
+}
+
+// Some styles in cloud-ui, interfere so adding additional selector to increase specificity
+body .hds-side-nav__header-logo {
+  &:hover,
+  &.hover {
+    background: var(--token-side-nav-surface-hover);
+    border-color: transparent;
+  }
+}
+
+body .hds-side-nav__header-logo {
+  &:active,
+  &.active {
+    background: var(--token-side-nav-surface-active);
+    border-color: transparent;
+  }
+}
+
+.hds-side-nav__header-logo-icon {
+  width: auto;
+  height: 35px;
+}
+
+// * Sub-component: Body
+// ---------------------------------------
+.hds-side-nav__body {
+  flex: 1;
+  padding: 12px;
+  overflow: auto;
+}
+
+.hds-side-nav__section {
+  color: var(--token-side-nav-foreground-primary); // Delete?
+  background: var(--token-side-nav-surface-default); // Q: Should this be added here or should it just be transparent?
+
+  + .hds-side-nav__section { margin-top: 28px; }
+}
+
+.hds-side-nav__section-title {
+  padding: 0 var(--token-side-nav-item-padding-horizontal);
+  color: var(--token-side-nav-foreground-faint);
+  background: var(--token-side-nav-surface-default); // Q: Should this be added here or should it just be transparent?
+
+  + .hds-side-nav__list { margin-top: 12px; }
+}
+
+.hds-side-nav__list {
+  margin: 0;
+  padding: 0;
+}
+
+.hds-side-nav__list-item {
+  list-style-type: none; // Q: Should ul have role="list" added for accessibility?
```

Comment:
should this declaration be moved to the `.hds-side-nav__list` directly?

Reply:
I added it on the `li` item because I thought that's what you normally preferred vs adding it to the `ul` which I normally prefer myself. I can change it.

Reply:
Mmm. Yeah, I see your point. Let's decide when we figure out how to split the `Item` vs `Link`

Reply:
I'll leave for now. Going to focus on making related changes to those at same time to try to avoid confusing myself.